### DISCUSSION TITLE
Aggregated Battery - how to set it up?

### DISCUSSION THREAD

**Discussion by seamaster101** [mr-manuel/venus-os_dbus-mqtt-battery#15] [General]
@mr-manuel you are mentioning that the driver can serve as an aggregated battery. I would like to explore this possibility and setup an aggregator to report the condition of my battery bank
I have 3 actual batteries that thanks to your driver and your help  now report properly (as a expression of my appreciation for the work you've done i bought you some beers yesterday :O). 
What is the next set to setup new instance of the driver that acts as an aggregator?

**Comment by mr-manuel** [comment]
You have to send the aggregated battery data over MQTT. This means that you need to aggregate the data somewhere else.

See https://github.com/mr-manuel/venus-os_dbus-mqtt-battery/discussions/10

Probably https://github.com/pulquero/BatteryAggregator or https://github.com/Dr-Gigavolt/dbus-aggregate-batteries could also work to aggregate the MQTT batteries.
